ui/components/Dataset/index.js

```diff
@@ -1,52 +1,70 @@
-import url from 'url';
-import Link from 'next/link';
-import { useRouter } from 'next/router';
-import { Snippet, Tag, Flex, Pagination } from '../';
-import upperFirst from 'lodash/upperFirst';
-import format from 'date-fns/format';
-import parseISO from 'date-fns/parseISO';
-import styles from './style.module.scss';
+import Link from "next/link";
+import { useRouter } from "next/router";
+import { Snippet, Tag, Flex, Pagination } from "../";
+import upperFirst from "lodash/upperFirst";
+import format from "date-fns/format";
+import parseISO from "date-fns/parseISO";
+import styles from "./style.module.scss";
 
-const DATE_FORMAT = 'do MMM yyyy';
+const DATE_FORMAT = "do MMM yyyy";
 
 function Model({ model, includeType }) {
+  console.log(model.name, model);
   const target = `/standards/model/${model.name}`;
-  const status = (model.extras.find(e => e.key === 'status') || {}).value;
-  const type = (model.extras.find(e => e.key === 'category') || {}).value || 'Uncategorised';
+  const status =
+    model && model.extras
+      ? model.extras.find((e) => e.key === "status").value
```

Comment:
I'm not going to go thru and suggest changing `"` for `'` everywhere - we generally use `'` for strings, `"` only for jsx props. Happy to be convinced otherwise, as long as we are consistent